File: mario.cpp
#include<stdio.h>

int main (void){
	int altura, cont,cont1,cont2, x, y;
	
	printf("Altura?  ");
	scanf("%d", &altura);
	x=1;
	for (cont=0; cont<altura; cont++){
		y=altura-x;
		for (cont2=0; cont2<y; cont2++)
			printf(" ");
		for (cont1=0; cont1<x; cont1++){
			printf("#");
		}
		printf(" ");
		for (cont1=0; cont1<x; cont1++){
			printf("#");
		}
		x++;
		printf("\n");
	}
	
	return 0;
}

File: mario.py

altura = int(input("Altura: "))
x=1
for i in range(0,altura,1):
    y=altura-x
    for i in range(0,y,1):
      print(" ",end="")
    for i in range(0, x, 1):
        print("#",end="")
    x += 1
    print()
